sopel/cli/run.py

```diff
@@ -385,7 +401,11 @@ def get_running_pid(filename):
 
 
 def command_start(opts):
-    """Start a Sopel instance"""
+    """Start a Sopel instance
+
+    :param opts: parsed arguments
+    :type opts: ``argparse.Namespace``
```

Comment:
This goes for all the occurrences of this class: Is there no easy cross-reference for use with the Sphinx `:class:` role?

Reply:
I may have forgot that for some reason. (probably a lack of sleep)